modules/osdk-ensuring-operator-workloads-run-restricted-psa.adoc

```diff
@@ -10,8 +10,8 @@ If you ensure your Operator workloads can run in namespaces set to the `restrict
 
 [WARNING]
 ====
-* You must leave the `runAsUser` field empty. If your image requires a user, ensure the `userID` is defined in the image and not by the security context.
 * Use a workload resource, such as `Deployment` or `StatefulSet`, to manage your Operator's pod. If your Operator creates workload pods by using an account with escalated SCC privileges, the pods might fail.
+* You must leave the `runAsUser` field empty. If your image requires a user, it cannot be run under security context constraints (SCC) and restricted pod security enforcement.
```

Comment:
> it cannot be run under security context constraints (SCC)

it cannot be run under restricted security context constraints (SCC)

Reply:
Addressed in 6946ad6